src/Domain/CoordinateMaps/TimeDependent/RotScaleTrans.hpp

```diff
@@ -28,52 +28,55 @@ namespace domain::CoordinateMaps::TimeDependent {
+ * and translation based on which maps are supplied.
  *
  * \details This map adds a rotation, expansion and translation based on what
  * types of maps are needed. Translation and expansion have piecewise functions
  * that map the coordinates $\vec{\xi}$ based on what region
  * $|\vec{\xi}|$ is in. Coordinates within the inner radius are translated by
- * the translation function of time T(t) and expanded by the inner expansion
- * function of time E_{a}(t). Coordinates in between the inner radius and outer
- * radius have a linear radial falloff applied to them. Coordinates at or beyond
- * the outer radius have no translation applied and are expanded by the outer
- * expansion function of time E_{b}(t). This map assumes that the center of the
- * map is at (0., 0., 0.). If translation or expansion is specified, then an
- * inner and outer radius must be specified as well. There is a rigid flag
- * which when set to true treats the coordinates as if they're within the inner
- * radius. This can be used as an optimization if you know which blocks in the
- * domain will always be within the inner radius.
+ * the translation function of time $\vec{T}(t)$ and expanded by the inner
+ * expansion function of time $E_{a}(t)$. Coordinates in between the inner
+ * radius and outer radius have a linear radial falloff applied to them.
+ * Coordinates at or beyond the outer radius have no translation applied and are
+ * expanded by the outer expansion function of time $E_{b}(t)$. This map assumes
+ * that the center of the map is at (0., 0., 0.). If translation or expansion is
+ * specified, then an inner and outer radius must be specified as well. There is
+ * a rigid flag which when set to true treats the coordinates as if they're
+ * within the inner radius. This can be used as an optimization if you know
+ * which blocks in the domain will always be within the inner radius.
+ *
+ * \note $E_{a}(t)$ and $E_{b}(t)$ are $a(t)$ and $b(t)$ from the CubicScale
+ * documentation.
  *
  * ## Mapped Coordinates
  * The RotScaleTrans map takes the coordinates $\vec{\xi}$ to the target
  * coordinates $\vec{\bar{\xi}}$ through
  * \f{equation}{
- * \vec{\bar{\xi}} = \left\{\begin{array}{ll}R(t)E_{a}(t)\vec{\xi} + \vec{T}(t),
- * & |\vec{\xi}| \leq R_{in}, \\ R(t)(w_{E} + E_{a}(t))\vec{\xi} + T(t) +
- * w_{T}T(t), & R_{in} < |\vec{\xi}| < 0.9 R_{out}, \\ R(t)(w_{E} +
- * E_{b}(t))\vec{\xi} + w_{T}T(t), & 0.9 R_{out} < |\vec{\xi}| < R_{out},
- * \\ R(t)E_{b}(t)\vec{\xi}, &
- * |\vec{\xi}| \geq R_{out} \end{array}\right.
+ * \vec{\bar{\xi}} = \left\{\begin{array}{ll}E_{a}(t)R(t)\vec{\xi} + \vec{T}(t),
+ * & |\vec{\xi}| \leq R_{in}, \\ (w_{E} + E_{a}(t))R(t)\vec{\xi} + (1 +
+ * w_{T})\vec{T}(t), & R_{in} < |\vec{\xi}| \leq 0.9 R_{out}, \\ (w_{E} +
+ * E_{b}(t))R(t)\vec{\xi} + w_{T}\vec{T}(t), & 0.9 R_{out} < |\vec{\xi}| <
+ * R_{out}, \\ E_{b}(t)R(t)\vec{\xi}, & |\vec{\xi}| \geq R_{out}
+ * \end{array}\right.
```

Comment:
[Discussed offline] Change all the factors to be $0.5 (R_{in} + R_{out})$ so we are guaranteed to be within the region. Using $0.5 R_{out}$ doesn't account for where the inner radius is.